atlassian_jwt_authentication.gemspec

```diff
@@ -3,7 +3,7 @@ require 'atlassian-jwt-authentication/version'
 
 Gem::Specification.new do |s|
   s.platform     = Gem::Platform::RUBY
-  s.name         = 'atlassian-jwt-authentication'
+  s.name         = 'atlassian-jwt-authentication-herocoders'
```

Comment:
would that be enough? I'm not familiar with the gem publishing process.

btw: who - from us, Herocoders - have sufficient permissions to publish this gem on rubygems.org? Do we need to setup something for our xyz@herocoders.com accounts?

Reply:
I don't remember, please check `.github/workflows/gem-push.yml`

Reply:
Hm... the script we use basically runs
```
          bundle exec rake build
          bundle exec rake release
```
So I guess changing `s.name` could be sufficient, WDYT?

I'm guessing that this site: https://rubygems.org/gems/atlassian-jwt-authentication is created according to the `s.name` - what else it could take? Folder name? Repo name? Doesn't make sense to me...

ref: https://guides.rubygems.org/specification-reference/#name